release_readme.rst

```diff
@@ -28,15 +28,17 @@ Package Contents
 Overview
 --------
 
-The F5® OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® load balancing services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
+The F5 OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® Local
+Traffic Manager™ (LTM®) services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
 
 Before You Begin
 ----------------
 
-You will need the following to use the F5® OpenStack LBaaSv1 plugin.
+You will need the following to use the F5 OpenStack LBaaSv1 plugin.
+
+-  Licensed BIG-IP (hardware or virtual edition)
+-  OpenStack |openstack| Neutron network deployment
```

Comment:
The change on line 40 should be reverted.  For LBaaSv1, releases from the 1.0 branch support "Icehouse - Kilo" and not just a single openstack version.